src/GeneralStateTestsFiller/stShift/shr_2^255_257Filler.json

```diff
@@ -0,0 +1,109 @@
+        "_info" : {
+            "commment" : "Taken from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-145.md"
+        },
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : ["<=Byzantium"],
+                "result" : {
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "balance" : "1000000000000000000",
+                        "storage" : {
+                            "0x00" : "0x03"
+                        }
+                    },
+                    "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                        "balance" : "400000",
+                        "code" : "0x",
+                        "nonce" : "0",
+                        "storage" : {
+                        }
+                    },
+                    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "balance" : "999999999999600000",
+                        "code" : "0x",
+                        "nonce" : "1",
+                        "storage" : {
+                        }
+                    }
+                }
+            },
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : [">=Constantinople"],
+                "result" : {
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "balance" : "1000000000000100000",
+                        "storage" : {
+                            "0x00" : "0x00"
+                        }
+                    },
+                    "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                        "balance" : "13006",
+                        "code" : "0x",
+                        "nonce" : "0",
+                        "storage" : {
+                        }
+                    },
+                    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                        "code" : "0x",
+                        "nonce" : "1",
+                        "storage" : {
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                "balance" : "1000000000000000000",
+                "code" : "PUSH32 0x8000000000000000000000000000000000000000000000000000000000000000; PUSH2 0x0101; SHR; PUSH1 0; SSTORE",
```

Comment:
comments //

Reply:
some json parsers eroors if file contains multimple fields with the same name

Reply:
I modified these duplicate keys.